apps/api-gateway/src/chat/chat.controller.ts

```diff
@@ -1,15 +1,31 @@
-import { Controller, Post, Body } from '@nestjs/common';
+import { Controller, Post, Body, Get, Param, Query } from '@nestjs/common';
 import { ApiTags } from '@nestjs/swagger';
 import { ChatService } from './chat.service';
-import { CreateChatRoomDto } from './dto/req.dto';
+import { CreateChatRoomDto, GetChatListDto, GetChatRoomDto, GetChatRoomsDto } from './dto/req.dto';
 
 @Controller('chat')
 @ApiTags('Chat API')
 export class ChatController {
   public constructor(private readonly chatService: ChatService) {}
 
-  @Post('create-chat-room')
+  @Post('chat-rooms')
   public createChatRoom(@Body() createChatRoomDto: CreateChatRoomDto) {
     return this.chatService.createChatRoom(createChatRoomDto.name);
   }
+
+  @Get('chat-rooms')
+  public getChatRooms(@Body() { id }: GetChatRoomsDto) {
+    //! id값은 userId로 인증/인가 절차 구현시 삭제 예정
+    return this.chatService.getChatRooms(id);
+  }
+
+  @Get('chat-rooms/:id')
+  public getChatRoom(@Param() { id }: GetChatRoomDto) {
+    return this.chatService.getChatRoom(id);
+  }
+
+  @Get('chat-lists/:id')
+  public getChatList(@Param() { id }: GetChatListDto, @Query() { page }: GetChatListDto) {
+    return this.chatService.getChatList(id, page);
+  }
```

Comment:
```suggestion
  @Get('rooms')
  public getChatRooms(@Body() { id }: GetChatRoomsDto) {
    //! id값은 userId로 인증/인가 절차 구현시 삭제 예정
    return this.chatService.getChatRooms(id);
  }

  @Get('rooms/:id')
  public getChatRoom(@Param() { id }: GetChatRoomDto) {
    return this.chatService.getChatRoom(id);
  }

  @Get('lists/:id')
  public getChatList(@Param() { id }: GetChatListDto, @Query() { page }: GetChatListDto) {
    return this.chatService.getChatList(id, page);
  }
```

지금보니까, chat/ 이 해당 서비스의 부모 Endpoint 여서

'chat-rooms' => 'rooms' 로 표현해줘도 충분하지 않을 까 싶어요!

Reply:
저번에 말씀 드린 것 처럼 chat room / room 중에 통일하기로 해서 chat-room으로 모두 통일하긴 했습니다!

Reply:
근데 현재 해당 API 에 대한 Endpoint 가
`api/chat/chat-rooms` 인데

'chat' 이라는 단어가 2번 들어갈 필요가 없지 않나 싶습니다

`api/chat/rooms` 로 충분히 Endpoint 정보를 표현할 수 있지 않을까요?

Reply:
chat api 한정해서는 그래도 좋을 것 같습니다

Reply:
저도 중복 맥락을 제거해주는 건 좋다고 생각합니다~

Reply:
네 `chat` 단어는 빼도 좋을 것 같습니다! 반영하겠습니다.